httpx/_models.py

```diff
@@ -441,7 +441,7 @@ def __init__(self, *args: QueryParamTypes, **kwargs: typing.Any) -> None:
             items = parse_qsl(value)
         elif isinstance(value, QueryParams):
             items = value.multi_items()
-        elif isinstance(value, list):
+        elif type(value) in [list, tuple]:
```

Comment:
In theory any `Sequence` of 2-tuples would do. We _could_ consider switching to `isinstance(value, typing.Sequence)`. But hmm, I don't think there's much of a case for anything else than lists and tuples, so this sounds good enough.

Reply:
I'm also in favor of using the more generic `Sequence`, makes the docs easier to change as well.

Reply:
Hey, sorry for the ignorance of mypy, but this error message seems to make no sense to me:

httpx/_models.py:445: error: Incompatible types in assignment (expression has type "Union[**Sequence[Tuple[str, Union[str, int, float, bool, None]]]**, str, bytes]", variable has type "**Sequence[Tuple[str, Union[str, int, float, bool, None]]]**")

Does anyone have an idea what I am doing wrong here?

Reply:
As per `isinstance(value, Sequence)` - it is a bit too broad. For example - String is also isinstance of Sequence:
```
>>> isinstance('asfd', Sequence)
True
```